web-app/js/ng-app/book/bookDetailsController.js

```diff
@@ -0,0 +1,13 @@
+angular.module('ang').controller('bookDetailsController', [
+   '$scope',
+   '$routeParams',
+   'bookService',
+   bookDetailsController
+]);
+
+function bookDetailsController($scope, $routeParams, bookService) {
+
+    bookService.getBookById($routeParams.id).then(function (response) {
+        $scope.book = response.data;
+    });
+}
```

Comment:
If you have more than one controller, or service, or filter per folder, move them to separate folder
"book/controllers", "book/filters" etc.